takeshape/penny
# Paginate product pages during site generation

### Test Plan (Steps to test):

1. Checkout this branch locally and run `npm run build`
1. Verify that the console output and the contents of `.next/server/pages/product` reflect 54 pages being generated

### Checklist:
